docs/tutorials/upgrade-a-chain/sudo-upgrade.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Use the Sudo Pallet
+---
+
+As the name of the [Sudo pallet](../../knowledgebase/runtime/frame#sudo) implies, it provides
+capabilities related to the management of a single
+[`sudo` ("superuser do")](https://en.wikipedia.org/wiki/Sudo) administrator. In FRAME, the `Root`
+Origin is used to identify the runtime administrator; some of FRAME's features, including the
+ability to update the runtime by way of
+[the `set_code` function](https://substrate.dev/rustdocs/v2.0.0-rc6/frame_system/enum.Call.html#variant.set_code),
+are only accessible to this administrator. The Sudo pallet maintains a single
+[storage item](../../knowledgebase/runtime/storage): the ID of the account that has access to the
+pallet's [dispatchable functions](../../knowledgebase/getting-started/glossary#dispatch). The Sudo
+pallet's `sudo` function allows the holder of this account to invoke a dispatchable as the `Root`
+origin. The following pseudo-code demonstrates how this is achieved, refer to the
+[Sudo pallet's source code](https://github.com/paritytech/substrate/blob/v2.0.0-rc6/frame/sudo/src/lib.rs)
+to learn more.
+
+```rust
+fn sudo(origin, call) -> Result {
+  // Ensure caller is the account identified by the administrator key
+  let sender = ensure_signed(origin)?;
+  ensure!(sender == Self::key(), Error::RequireSudo);
+
+  // Dispatch the specified call as the Root origin
+  let res = call.dispatch(Origin::Root);
+  Ok()
+}
+```
+
+## Start the Template Node
+
+Since forkless runtime upgrades do not require network participants to restart their blockchain
+clients, the first step of this tutorial is to start the template node as-is. Build and start the
+unmodified [Node Template](https://github.com/substrate-developer-hub/substrate-node-template).
+
+```shell
+cargo run --release -- --dev --tmp
```

Comment:
For a tutorial do we really need users to run their node in `release` mode? It's just going to add extra time to their compilation. I know that `set_code` can take a long time, but I imagine that for a local chain it'll be fine with no optimizations

Reply:
This is a really great comment, @HCastano :pray: I kind of always just took for granted that all our docs specified a release build. I just did [a little research](https://users.rust-lang.org/t/why-does-cargo-build-not-optimise-by-default/4150/2) and it completely makes sense that we'd want to omit this flag in tutorials and such.

Reply:
I thought that compiling without optimizations will result in the binary not even able to produce blocks? It's really only good for checking if it does compile.

Reply:
I definitely checked :+1: I can confirm that the node produces and finalizes blocks :heavy_check_mark:

Reply:
> It's really only good for checking if it does compile.

@joepetrowski There's a special cargo command for this, [`cargo check`](https://doc.rust-lang.org/cargo/commands/cargo-check.html), which doesn't build a binary but just runs through part of the compilation process to make sure your code compiles